datafusion/sql/src/unparser/expr.rs

```diff
@@ -347,7 +585,7 @@ mod tests {
 
         let actual = format!("{}", ast);
 
-        let expected = r#"'a' > 4"#;
+        let expected = r#"('a' > 4)"#;
```

Comment:
these parens are due to using `Expr::Nested` right? I think it is a good design to add explicit parens to ensure the evaluation order is correctly reflected

Reply:
I'm not sure what you mean be explicit parens... is that different than ast::Expr::Nested? I thought that Nested was how the AST represents explicit parenthesis.